cmd/server.go

```diff
@@ -0,0 +1,31 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/gravitl/netclient/functions"
+	"github.com/spf13/cobra"
+)
+
+// switchServer command is used to set netclient context to a server, it has registered already.
+var switchServer = &cobra.Command{
+	Use:   "switch [ servername ]",
+	Short: "switch [ servername ]",
+	Long:  `switches netclient to a registered server`,
+	Args:  cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		err := cobra.OnlyValidArgs(cmd, args)
```

Comment:
ValidArgs  has not been defined; won't this return an err every time?
actually it will return nil as len(cmd.ValidArgs) == 0 but this check is not doing anything